cognite/seismic/protos/v1/seismic_service.proto

```diff
@@ -114,6 +114,8 @@ service SeismicAPI {
      */
     rpc GetVolume(VolumeRequest) returns (stream com.cognite.seismic.Trace) {}
 
+    rpc GetVolumeBounds(VolumeRequest) returns (VolumeBoundsResponse) {}
```

Comment:
Should we put this in https://github.com/cognitedata/seismic-proto/blob/master/cognite/seismic/protos/experimental/experimental_service.proto instead and keep it there until we can graduate it to support a new StreamTraces API?

Reply:
Sure, we could. But the sdk will use it, so not sure how 'experimental' it's going to be :smile:

Reply:
Also, if we're going to keep GetVolume around forever, why not just keep this around as well? :upside_down_face: